taca/nanopore/instrument_transfer.py

```diff
@@ -69,6 +75,50 @@ def main(args):
             final_sync_to_storage(run_path, rsync_dest, args.archive_dir, rsync_log)
 
 
+def delete_archived_runs(args):
+    logging.info("Finding locally archived runs...")
+    # Look for dirs matching run pattern inside archive dir
+    run_paths = [
+        path
+        for path in glob(os.path.join(args.archive_dir, "*", "*", "*"), recursive=True)
+        if re.match(RUN_PATTERN, os.path.basename(path))
+    ]
+    logging.info(f"Found {len(run_paths)} locally archived runs...")
+
+    preproc_archive_contents = os.listdir(os.path.join(args.dest_dir, "nosync"))
```

Comment:
From what I've heard the archiving is in fact automated so maybe using the dir for archived runs would be even safer (and slower...). We can perhaps hear what the others say if it's ever happened that it's not reliable to trust the nosync location.